front/pages/api/poke/workspaces/[wId]/data_sources/[name]/index.ts

```diff
@@ -7,6 +8,8 @@ import { ReturnedAPIErrorType } from "@app/lib/error";
 import { DataSource, Workspace } from "@app/lib/models";
 import { apiError, withLogging } from "@app/logger/withlogging";
 
+const { DUST_DATA_SOURCES_BUCKET = "", SERVICE_ACCOUNT } = process.env;
```

Comment:
Why `= ""` for `DUST_DATA_SOURCES_BUCKET` but not for `SERVICE_ACCOUNT` ?

Reply:
Cause I copy pasted from cli.ts like a monkey 😅 
I added a commit to fix and raise an error if no `DUST_DATA_SOURCES_BUCKET`